frontend/src/components/page-about/index/partners.js

```diff
@@ -16,7 +17,7 @@ const Partners = () => (
   <div className={container}>
     <div className={globalMissionContainer}>
       <div>
-        <img src={hmiImage} alt="HMI Project" />
+        <StaticImage src="../../../images/image264.png" alt="HMI Project" />
```

Comment:
This lost it spacing between image and global content beneath the image: 
![image](https://github.com/hmcc-global/hmccaa-web/assets/9808478/9267496e-91c4-4710-9e0c-cd79233aa95c)